src/de/app/mod.rs

```diff
@@ -29,49 +27,48 @@ impl<'de> Deserialize<'de> for CommandWrap<'de> {
     }
 }
 
-struct CommandVisitor<'a>(Command<'a>);
+struct CommandVisitor(Command);
 
-impl<'a> Visitor<'a> for CommandVisitor<'a> {
-    type Value = CommandWrap<'a>;
+impl<'de> Visitor<'de> for CommandVisitor {
+    type Value = CommandWrap;
 
     fn expecting(&self, formatter: &mut std::fmt::Formatter) -> std::fmt::Result {
         formatter.write_str("Command Map")
     }
 
     fn visit_map<A>(self, mut map: A) -> Result<Self::Value, A::Error>
     where
-        A: serde::de::MapAccess<'a>,
+        A: serde::de::MapAccess<'de>,
     {
         let mut app = self.0;
         //TODO: check the first key to get name from the input?
         //currently the name change in `Clap::Command::name` doesn't change the `Clap::Command::id` so might cause problems?
         while let Some(key) = map.next_key::<&str>()? {
             app = parse_value!(key, app, map, Command, {
-                (about, &str),
-                (after_help, &str),
-                (after_long_help, &str),
-                (alias, &str),
-                ref (aliases, Vec<&str>),
+                (about, String),
+                (after_help, String),
+                (after_long_help, String),
```

Comment:
For your use case, does it matter if these help values are `StyledStr`?

(ditto for all of the others)